src/main/java/com/google/devtools/build/lib/actions/cache/CompactPersistentActionCache.java

```diff
@@ -466,6 +467,14 @@ private static void encodeRemoteMetadata(
     VarInt.putVarInt(value.getLocationIndex(), sink);
 
     VarInt.putVarInt(indexer.getOrCreateIndex(value.getActionId()), sink);
+
+    Optional<PathFragment> symlinkTargetExecPath = value.getSymlinkTargetExecPath();
```

Comment:
Please update `VERSION` number in this file so that we won't read old content.

Reply:
Done.